apps/nextjs-website/src/components/atoms/GuideMenu/GuideMenu.tsx

```diff
@@ -149,17 +153,26 @@ const GuideMenu = ({
 }: GuideMenuProps) => {
   const { palette } = useTheme();
   const { shared } = translations;
-
+  const scrollUp = useScrollUp();
   const currentPath = usePathname();
   const segments = currentPath.split('/');
   const expanded = segments.map((_, i) => segments.slice(0, i + 1).join('/'));
+  const top = scrollUp
+    ? SITE_HEADER_HEIGHT + PRODUCT_HEADER_HEIGHT
+    : PRODUCT_HEADER_HEIGHT;
   return (
     <Box
       sx={{
         backgroundColor: palette.grey[50],
         display: 'flex',
         flexDirection: 'column',
-        position: 'relative',
+        flexShrink: 0,
+        position: { lg: 'sticky' },
+        top: { lg: top },
+        height: { lg: `calc(100vh - ${top}px)` },
+        overflowY: 'auto',
+        transition: 'all 0.5s linear',
+        scrollbarWidth: 'thin',
```

Comment:
When the scrollbar appears the width of the menu changes and the content on the right shifts. I think we should use something like [scrollbar-gutter](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter#example_3) to keep the width of the column consistent.

[screencast-localhost_3000-2023.11.14-12_10_27.webm](https://github.com/pagopa/developer-portal/assets/39835990/e29af492-9f5c-4554-8d34-4bb5cf4a2503)